jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/ResourceFactoryTest.java

```diff
@@ -404,6 +404,39 @@ public void testSplitOnSemicolon()
         }
     }
 
+    @Test
+    public void testSplitOnPathSeparatorWithGlob() throws IOException
+    {
+        try (ResourceFactory.Closeable resourceFactory = ResourceFactory.closeable())
```

Comment:
Could you add another term to the glob string that is just a single jar file, to mimic a classpath that has both glob _and_ individual jar files on it.

Reply:
This test kind has that already, 2 non globs, and 1 glob.
Also the existing test `testSplitOnPipeWithGlob()` has the same.
Do you want the non-glob to be specifically a jar?

Reply:
Followup, do you want the extra non-globs to be before or after the glob?

Reply:
I don't really mind the position of the jar, just so long as the test is a combination of dirs, a dir with glob and at least one explicit jar.